ComfortZoneApp/app/src/main/java/com/example/comfortzone/notification/NotificationUtil.java

```diff
@@ -41,28 +40,41 @@ public class NotificationUtil {
     public static final int DEFAULT_HOUR = 10;
     public static final int DEFAULT_MIN = 0;
 
-    public static void notificationSetup(Context context) {
-        Calendar calendar = Calendar.getInstance();
-        calendar.set(Calendar.HOUR_OF_DAY, NOTIF_TIME);
-        calendar.set(Calendar.MINUTE, 0);
-        calendar.set(Calendar.SECOND, 0);
-
-        if (calendar.getTime().compareTo(new Date()) < 0)
-            calendar.add(Calendar.DAY_OF_MONTH, 1);
+    private static AlarmManager getAlarmManager(Context context) {
+        return (AlarmManager) context.getSystemService(ALARM_SERVICE);
```

Comment:
is a separate method needed? I think for this one you can just call the getSystemService directly wherever you need the AlarmManager